docs/usage/etcd_encryption_config.md

```diff
@@ -0,0 +1,25 @@
+# ETCD Encryption Config
+
+The `spec.kubernetes.kubeAPIServer.encryptionConfig` field in the Shoot API allows users to customize encryption configurations for the API server. It provides options to specify additional resources for encryption beyond secrets.
+
+## Usage Guidelines
+
+- The `resources` field can be used to specify resources that should be encrypted in addition to secrets. Secrets are always encrypted.
+- Each item is a Kubernetes resource name in plural (resource or resource.group). Wild cards are not supported.
+- Adding an item to this list will cause empty patch requests for all the resources of that kind to encrypt them in the etcd. See [Encrypting Confidential Data at Rest](https://kubernetes.io/docs/tasks/administer-cluster/encrypt-data) for more details.
```

Comment:
No, there is no labelling involved.

Reply:
Why not? The motivation of the labelling was to save sending updates for already encrypted/decrypted resources again in case the reconciliation fails intermittently.